.github/actions/make/uniffi-bindgen/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Fetch or make uniffi-bindgen binary
+
+runs:
+  using: composite
+  steps:
+    - name: uniffi bindgen artifact name
+      run: echo "ARTIFACT_NAME=uniffi-$(uname -s)-$(cargo info uniffi | grep version | head -1 | awk '{print $2}')" >> $GITHUB_ENV
```

Comment:
I'd break up those sub invocations. Also, `cargo metadata` may be more stable over time than `cargo info`; it's intended for scripting purposes.

```sh
os="$(uname -s)"
uniffi_version="$(cargo metadata --format-version 1 | jq -r '.packages[] | select(.name == "uniffi") | .version'
echo "ARTIFACT_NAME=uniffi-${os}-${uniffi-version}" >> $GITHUB_ENV
```